spec/ics-003-connection-semantics/README.md

```diff
@@ -294,7 +294,6 @@ A specific version can optionally be passed as `version` to ensure that the hand
 ```typescript
 function connOpenInit(
   identifier: Identifier,
```

Comment:
should this not be created by `generateIdentifier`?

Reply:
Yes, it should - fixed.